wolfcrypt/benchmark/benchmark.c

```diff
@@ -463,10 +463,15 @@
     #define TEST_SLEEP() WC_DO_NOTHING
 #endif
 
-#define TEST_STRING    "Everyone gets Friday off."
-#define TEST_STRING_SZ 25
-
 
+#if defined(WOLFSSL_MICROCHIP_TA100)
+    /* pad test message to 32 bytes */
+    #define TEST_STRING    "Everyone gets Friday off.padto32"
```

Comment:
The issue here is the TA100 is expecting the hash algo SHA2-256 of 32-bytes. Your solution here is wrong. If the RSA sign expect a hash then we should use a hash...

Reply:
@dgarske, TA100 signing necessitates a hash input. To meet this requirement, I've adjusted the plaintext to match the hash size, eliminating the need to factor in software-generated hashing time. This was done on purpose.

Reply:
You are doing this wrong. That's fine that the TA100 requires the input to to match the hash algo. We should be zero padding or truncating in our ecc.c for TA100, not on the test case. That's what we do in wc_ecc_sign_hash_ex. See `/* load digest into e */`.